src/components/v2/values/common/ChartValuesSelectors.tsx

```diff
@@ -456,14 +456,21 @@ export const ChartValuesEditor = ({
     useEffect(() => {
         // scroll to the editor view with animation for only update-chart
         // subtracting - 100 from offset top because of floating header's tab
-        if (autoFocus && parentRef?.current && editorRef?.current) {
-            setTimeout(() => {
-                parentRef.current.scrollTo({
-                    top: editorRef.current.offsetTop - 100,
+        let timer;
```

Comment:
This file's change is not related to this story. I hope this is intentional and properly reviewed

Reply:
It's to fix an issue observed during testing. The issue is something to do with setTimeout functionality that occurs when a user keeps switching between tabs quickly/rapidly.